nncf/torch/composite_compression.py

```diff
@@ -144,3 +144,8 @@ def apply_to(self, target_model: NNCFNetwork) -> NNCFNetwork:
         for ctrl in self.child_ctrls:
             target_model = ctrl.apply_to(target_model)
         return target_model
+
+    def load_state(self, states):
+        self._check_loaded_compression_stage(states)
+        for child_ctrl, child_state in zip(self.child_ctrls, states['scheduler']):
+            child_ctrl.load_state({'scheduler': child_state})
```

Comment:
@vanyalzr Have you updated the checkpoints according to these changes?

Reply:
PyTorch checkpoints with the composite compression algorithm already had the `scheduler` field as a list, it's just that this list is now in `states['scheduler']`, not in just `states`, hence no need to update checkpoints here